test/app/features/appstate/domain/usecases/app_preferences_usecase_test.dart

```diff
@@ -238,10 +270,9 @@ void main() {
       );
 
       expect(value.isLeft(), true);
-      expect(
-        value.swap().getOrElse(() => null),
-        InactivityError.customerNotStealth,
-      );
+      value.swap().forEach((error) {
+        expect(error, InactivityError.customerNotStealth);
+      });
```

Comment:
```suggestion
      expect(
        value.fold((l) => l, (r) => r),
        InactivityError.customerNotStealth,
      );
```

Acredito que utilizar o `.fold` facilita a leitura do teste.

Reply:
feito